lox_services/scraping/selenium_util.py

```diff
@@ -169,7 +199,7 @@ def wait_then_clear(
         - `wait`: The wait element that will be used to wait until a certain number of second.
         - `selector`: The selector used to look for the elements.
     """
-    wait.until(lambda driver: driver.find_element(By.CSS_SELECTOR, selector))
+    wait.until(lambda dr: dr.find_element(By.CSS_SELECTOR, selector))
```

Comment:
please reverse this change

Reply:
Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes